modules/setting/security.go

```diff
@@ -157,4 +158,7 @@ func loadSecurityFrom(rootCfg ConfigProvider) {
 			PasswordComplexity = append(PasswordComplexity, name)
 		}
 	}
+
+	// TODO: default value should be true in future releases
```

Comment:
```suggestion
	// TODO: default value should be true after release v1.22
```

Reply:
Alternatively,
```suggestion
	// TODO: default value should be true immediately after 1.22.0 has been released, so that 1.23.0 ships with the change
```